metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -536,11 +602,17 @@ def generate_dataset_profile(  # noqa: C901 (complexity)
                 column_profile.nullCount = null_count
```

Comment:
I think we need to revise the above assert statement. If non_null_count > row_count, then null_count < 0, and we should simply reset it to 0 instead of throwing an assertion error.

Reply:
I updated this as well.